docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-product-feature-integration.md

```diff
@@ -146,3 +144,15 @@ Make sure that when you do a `GET` request for the carts with merchant products,
 Make sure that when you do a `GET` request to retrieve abstract products that belong to a specific merchant, it returns products' data together with their merchants `http://glue.de.demo-spryker.com/abstract-products/{% raw %}{{abstractProductSku}}{% endraw %}?include=merchants`.
 
 {% endinfo_block %}
+
+
+### 4) Add translations
+
+```
+merchant_product.message.invalid,Product "%sku%" with Merchant "%merchant_reference%" not found.,en_US
+merchant_product.message.invalid,Der Produkt "%sku%" mit dem Händler "%merchant_reference%" ist nicht gefunden.,de_DE
+merchant.message.removed,Merchant "%merchant_reference%" not found.,en_US
+merchant.message.removed,Händler "%merchant_reference%" ist nicht gefunden.,de_DE
+merchant.message.inactive,Merchant "%merchant_reference%" is not active.,en_US
+merchant.message.inactive,Der Händler "%merchant_reference%" ist nicht aktiv.,de_DE
```

Comment:
Part of `Merchant` module which doesn't belong to this feature. Please remove.

Reply:
True. Removed.